src/Ledger/Ratify.lagda

```diff
@@ -249,7 +427,19 @@ delayingAction Info                 = false
 delayed : (a : GovAction) → NeedsHash a → EnactState → Bool → Set
 delayed a h es d = ¬ verifyPrev a h es ⊎ d ≡ true
 \end{code}
+} %% end small
+\caption{%Ratify iv:
+Determination of the status of ratification of the governance action}
+\label{fig:defs:ratify-iv}
 \end{figure*}
+The code in Figure~\ref{fig:defs:ratify-iv} defines still more types required for ratification of a governance action.
+\begin{itemize}
+  \item \verifyPrev takes the given governance action, its \NeedsHash, and \EnactState record and checks whether
+  the given governance action is already verified as one that can be applied in the current state;
+  \item \delayingAction takes a governance action and returns \true if it is a ``delaying action'' (\NoConfidence, \NewCommittee, \NewConstitution, \TriggerHF)
+  and returns \false otherwise;
+  \item \delayed checks whether a given \GovAction is delayed.
```

Comment:
This should actually get more explanation. An action can be delayed either because a previous action was a `delayingAction`, or because the hash check failed.